React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -556,12 +577,16 @@ - (void)_updateState
 
 - (AttributedString::Range)_selectionRange
 {
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
   UITextRange *selectedTextRange = _backedTextInputView.selectedTextRange;
   NSInteger start = [_backedTextInputView offsetFromPosition:_backedTextInputView.beginningOfDocument
                                                   toPosition:selectedTextRange.start];
   NSInteger end = [_backedTextInputView offsetFromPosition:_backedTextInputView.beginningOfDocument
                                                 toPosition:selectedTextRange.end];
   return AttributedString::Range{(int)start, (int)(end - start)};
+#else
+  return AttributedString::Range({0, 1});
```

Comment:
If this is a placeholder, you should mark it as such and create an issue to implement it properly.

Reply:
Added comment. Will add issue once I discuss best format for the Fabric follow up work w/
 @Saadnajmi